.buildkite/platforms/linux64.yml

```diff
@@ -70,7 +70,10 @@ steps:
       julia-artifact/bin/julia -e '@info "" Sys.CPU_THREADS'
 
       echo "--- Run the Julia test suite"
-      julia-artifact/bin/julia .buildkite/rr_capture.jl julia-artifact/bin/julia -e 'Base.runtests(["all"]; ncores = Sys.CPU_THREADS)'
+      
+      
+      
+      julia-artifact/bin/julia .buildkite/rr_capture.jl julia-artifact/bin/julia -e 'const HOME = joinpath(pwd(), "myhomedir"); mkpath(HOME); ENV["HOME"] = HOME; ENV["USER"] = "myusername"; Base.runtests(["all"]; ncores = Sys.CPU_THREADS)'
```

Comment:
I don't think you want to do it this way; instead you should just set `HOME: /home` in the YAML environment variable section.  That is an empty directory that should be fine to modify.

Reply:
Done!